tests/regression/echotest/ord-tests-expected.echo

```diff
@@ -1,26 +1,27 @@
 ECHO: 97
 ECHO: 97
+WARNING: ord() number of parameters does not match: expected 1, found 0 in file ord-tests.scad, line 9
 ECHO: undef
 ECHO: undef
-WARNING: ord() argument undef is not of type string, in file ord-tests.scad, line 11
+WARNING: ord() parameter could not be converted: argument 0: expected string, found undefined in file ord-tests.scad, line 11
 ECHO: undef
-WARNING: ord() argument inf is not of type string, in file ord-tests.scad, line 12
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 12
 ECHO: undef
-WARNING: ord() argument -inf is not of type string, in file ord-tests.scad, line 13
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 13
 ECHO: undef
-WARNING: ord() argument nan is not of type string, in file ord-tests.scad, line 14
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 14
 ECHO: undef
-WARNING: ord() argument 3.1416 is not of type string, in file ord-tests.scad, line 15
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 15
```

Comment:
Looks like this is not caused by your changes, but would you mind having a look if it's easily possible to get the actual value back into this message, that seems useful? Otherwise we can create a ticket so the topic is not lost.

Reply:
sure, I can look into it.

The merge request was partly to see if the CI-Test uncovers unintended side effects (it did not for now), to confirm that my expected files works on Windows and Mac OS X (it does) and in part to get feedback what actually is desired output of ord() and if my approach to workaround the issue is acceptable.

Reply:
Cool, thanks. I'm still puzzled why your pull requests don't trigger the CircleCI builds. I guess the test suite runs are more important, but still strange.

Reply:
> I'm still puzzled why your pull requests don't trigger the CircleCI builds. 
> I guess the test suite runs are more important, but still strange.
Did you notice that I still use my own fork on github and thus use "across forks"?

I think that github treads "across fork" merge request differently from merge request within a project. (Regardless of the user being a member or not)

I could push to OpenSCAD instead of my fork, but I do not know the precise implications.
(e.g. I remember it used to be that we had restriction on Mac OS X Builds)

Reply:
just as note for myself:

https://github.com/openscad/openscad/blob/4db149f831e7b1c4382b1110f64b8ce340ed8492/src/core/builtin_functions.cc#L396-L400

https://github.com/openscad/openscad/blob/4db149f831e7b1c4382b1110f64b8ce340ed8492/src/core/Parameters.cc#L241-L263

Reply:
a train of thought: toEchostring can generate an error and thus throw an exception, for e.g. when the to vector to string conversion exhausts the stack.

This effectively can turn a warning into an error.

But... people running into that quirk, like have long running script, that should use "stop at first warning".

More problematic is, that the failing toEchoStrings prevents the warning from being generated.
That can be a real problem, as that can easily happen, when two arguments in a call are swapped. Knowing that there is a type error is more helpful, than knowing that converting the vector would exhaust the stack (which might never happen during normal runtime).

Reply:
Yes, cross fork is different, but it's configured to still build just without the secret keys. It did work for example in #4215. Very strange.
Maybe it's confused by the combination of you having access to the openscad main repo and creating a pull request from the separate account. I'll try that with my account with the next change.